impl_core/src/test/java/io/opencensus/implcore/tags/propagation/TagContextSerializationTest.java

```diff
@@ -86,6 +87,17 @@ public void testSerializeWithMultipleTags() throws Exception {
     testSerialize(T1, T2, T3, T4);
   }
 
+  @Test
+  public void testSerializeTooLargeTagContext() throws TagContextSerializationException {
```

Comment:
Same here.  It would be useful to have one test that successfully serializes a TagContext that is at the size limit and another test that throws an exception when serializing a TagContext that is one byte larger.

Reply:
As above, added to `TagContextRoundtripTest`.

Reply:
The roundtrip test serializes a TagContext that is exactly the size limit, but it would also be useful to have a test that serializes a TagContext that is exactly one byte too large.  It looks like this test adds 8192 tags, which is much larger.

Reply:
Sure, I updated this to test against a TagContext with serialized size 8193.